packages/familie-header/src/søk/idnrValidator.ts

```diff
@@ -0,0 +1,60 @@
+const elevenDigits = new RegExp('^\\d{11}$')
+
```

Comment:
Should we return a typescript type or something similar?
```
type FnrValidationResult = "invalid" | "valid"
interface FnrValidation { // eller noe norskt
   status: FnrValidationResult
   type: ..
   resons: .. ? 
}
```

Reply:
I think true/false is enough